vtex-apps/zendesk-chat
# Add settings for button appearance

#### What is the purpose of this pull request?

Closes https://github.com/vtex-apps/store-discussion/issues/465

#### How should this be manually tested?

Configuration:
https://kiwi--storecomponents.myvtex.com/admin/apps/vtex.zendesk-chat@1.3.2/setup/

Home:
https://kiwi--storecomponents.myvtex.com/

Check the button bg color, text color, label and position.

#### Types of changes
